conda/recipes/pynvjitlink/recipe.yaml

```
@@ -0,0 +1,69 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.
+schema_version: 1
+
+context:
+  cuda_version: ${{ load_from_file("pynvjitlink/CUDA_VERSION") | trim }}
+  date_string: '${{ env.get("RAPIDS_DATE_STRING") }}'
+  head_rev: ${{ git.head_rev(".")[:8] }}
+  py_version: ${{ env.get("RAPIDS_PY_VERSION") }}
+  py_buildstring: ${{ py_version | version_to_buildstring }}
+  version: ${{ load_from_file("pynvjitlink/VERSION") | trim }}
+
+package:
+  name: pynvjitlink
+  version: ${{ version }}
+source:
+  path: ../../..
+
+build:
+  dynamic_linking:
+    overlinking_behavior: error
+  string: ${{ cuda_version }}_py${{ py_buildstring }}_${{ date_string }}_${{ head_rev }}
```

Can we add the package hash here too?

```suggestion
  string: ${{ cuda_version }}_py${{ py_buildstring }}_${{ date_string }}_${{ head_rev }}_h${{ PKG_HASH }}
```

ref: https://rattler.build/latest/build_script/#default-environment-variables-set-during-the-build-process

---

done and done!